simulation/simulation_exp3.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import sys
+sys.path.append("..")
+from striatum.storage import history
+from striatum.storage import model
+from striatum.bandit import exp3
+
+
+class Exp3LinearPayoff:
+
+    def __init__(self, t, d, actions):
+        self.T = t
```

Comment:
The uppercase t might not be a good choice for naming. Numpy transpose function is also uppercase t:
```python
import numpy as np

a = np.ones((10, 20))
b = a.T
```